src/__experimental__/components/search/search.spec.js

```diff
@@ -30,27 +30,44 @@ describe('Search', () => {
       }, renderWrapper({ value: '' }, mount));
     });
 
+    it('applies the default width when the user does not specify a width', () => {
+      assertStyleMatch({
+        borderBottom: '2px solid #CCD6DA',
+        display: 'inline-flex',
+        fontSize: '14px',
+        fontWeight: '700'
+      }, renderWrapper({ value: '', searchWidth: null }, mount));
+    });
+
+    it('applies the correct width specified by the user', () => {
+      assertStyleMatch({
+        borderBottom: '2px solid #CCD6DA',
+        display: 'inline-flex',
+        fontSize: '14px',
+        fontWeight: '700'
+      }, renderWrapper({ value: '', searchWidth: 400 }, mount));
+    });
+
     it('matches the expected styles when the input is focused', () => {
       wrapper = renderWrapper({ value: '' }, mount);
       const input = wrapper.find('input');
       input.simulate('focus');
       assertStyleMatch({
-        borderBottom: '2px solid transparent'
+        borderBottom: '2px solid #CCD6DA'
       }, wrapper);
     });
 
     it('matches the expected styles when the search is active', () => {
```

Comment:
```suggestion
    it('matches the expected styles when there is active and has a value', () => {
```

Reply:
Done. This has been changed to `'matches` the expected styles when the search is active and has a value'`